src/openklant/components/klantinteracties/api/serializers/internetaken.py

```diff
@@ -68,20 +71,43 @@ class Meta:
             },
         }
 
+    def validate(self, attrs):
+        status = attrs.get("status", None)
+        if not status and self.instance:
+            status = self.instance.status
+
+        if attrs.get("afgehandeld_op") and status == Taakstatus.te_verwerken.value:
```

Comment:
Minor, but I don't think you have to specifically compare to `.value`. Additionally, I'm not sure if new `Taakstatus` values will be added in the future, but to be sure I would probably check if the status is not `verwerkt` instead, to account for that if it does happen.
```suggestion
        if attrs.get("afgehandeld_op") and status != Taakstatus.verwerkt:
```